tests/containers/workbenches/rstudio/rstudio_tests.py

```diff
@@ -0,0 +1,155 @@
+
+if TYPE_CHECKING:
+    import docker.models.images
+
+
+class TestRStudioImage:
+    """Tests for RStudio Workbench images in this repository."""
+
+    @allure.issue("RHOAIENG-17256")
+    def test_rmd_to_pdf_rendering(self, image: str) -> None:
+        """
+        References:
+            https://stackoverflow.com/questions/40563479/relationship-between-r-markdown-knitr-pandoc-and-bookdown
+            https://www.earthdatascience.org/courses/earth-analytics/document-your-science/knit-rmarkdown-document-to-pdf/
+        """
+        skip_if_not_rstudio_image(image)
+
+        container = WorkbenchContainer(image=image, user=1000, group_add=[0])
+        try:
+            container.start(wait_for_readiness=False)
+
+            # language=R
+            script = textwrap.dedent("""
+                library(knitr)
+                library(rmarkdown)
+                render("document.Rmd", output_format = "pdf_document")
+                """)
+            # language=markdown
+            document = textwrap.dedent("""
+                ---
+                title: "Untitled"
+                output: pdf_document
+                date: "2025-01-22"
+                ---
+
+                ```{r setup, include=FALSE}
+                knitr::opts_chunk$set(echo = TRUE)
+                ```
+
+                ## R Markdown
+
+                This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+                When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+                ```{r cars}
+                summary(cars)
+                ```
+
+                ## Including Plots
+
+                You can also embed plots, for example:
+
+                ```{r pressure, echo=FALSE}
+                plot(pressure)
+                ```
+
+                Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+                """)
+
+            with tempfile.TemporaryDirectory() as tmpdir:
+                tmpdir = pathlib.Path(tmpdir)
+                (tmpdir / "script.R").write_text(script)
+                docker_utils.container_cp(container.get_wrapped_container(), src=str(tmpdir / "script.R"),
+                                          dst="/scripts")
+                (tmpdir / "document.Rmd").write_text(document)
+                docker_utils.container_cp(container.get_wrapped_container(), src=str(tmpdir / "document.Rmd"),
+                                          dst="/scripts")
+
+            # copy to a (writable) working directory
+            check_call(container, "bash -c 'cp /scripts/document.Rmd ./'")
```

Comment:
Instead of copying this again, wouldn't it be better to copy these files to the final directory right away?

Reply:
very good point, fixed it now
/override ci/prow/images